src/Symfony/Component/Validator/Constraints/NotJson.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Imad ZAIRIG <imadzairig@gmail.com>
+ */
+class NotJson extends Constraint
```

Comment:
```php
class Foo
{
    /** @Assert\NotJson */
    public $json;
}
```

When I put my constraint on here, I check if it's not json, but the validator will check the exact opposite. I think the correct name of this class should by something like `Json` (`Assert\Json`) or `IsJson` (`Assert\IsJson`).

Reply:
In the first commit it was IsJson and after a discussion with @onEXHovia , i've changed it to NotJson i think we can change it to only Json to be simple and more telling, what do you think ?

Reply:
I've changed the name to `Json.